.github/workflows/playwright.yml

```diff
@@ -0,0 +1,33 @@
+name: Playwright Tests
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+jobs:
+  test:
+    defaults:
+      run:
+        working-directory: ./prototypes/basic
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - uses: actions/setup-node@v4
+      with:
+        node-version: lts/*
+    - name: Install lib dependencies -- todo, check why
+      working-directory: ./lib/importer
+      run: npm i
+    - name: Install dependencies
+      run: npm i
+    - name: Install Playwright Browsers
+      run: npx playwright install --with-deps
+    - name: Run Playwright tests
+      run: npx playwright test
```

Comment:
Is it worth using [`--reporter=github`](https://playwright.dev/docs/next/test-reporters#github-actions-annotations) and/or [`--reporter=junit`](https://playwright.dev/docs/next/test-reporters#junit-reporter)?

Reply:
Trying it out and if I can get junit to write to test-results as well can fix two things at once.